inference-engine/tests/functional/plugin/cpu/test_utils/cpu_test_utils.cpp

```diff
@@ -76,6 +76,27 @@ std::string CPUTestsBase::impls2str(const std::vector<std::string> &priority) {
     return str;
 }
 
+const char *CPUTestsBase::cpu_prc2str(InferenceEngine::Precision v) {
+    if (v == InferenceEngine::Precision::FP64) return "FP64";
+    if (v == InferenceEngine::Precision::I64) return "I64";
+    if (v == InferenceEngine::Precision::U64) return "U64";
+    if (v == InferenceEngine::Precision::FP32) return "FP32";
+    if (v == InferenceEngine::Precision::I32) return "I32";
+    if (v == InferenceEngine::Precision::U32) return "U32";
+    if (v == InferenceEngine::Precision::BF16) return "BF16";
+    if (v == InferenceEngine::Precision::FP16) return "FP16";
+    if (v == InferenceEngine::Precision::I16) return "I16";
+    if (v == InferenceEngine::Precision::U16) return "U16";
+    if (v == InferenceEngine::Precision::I8) return "I8";
+    if (v == InferenceEngine::Precision::U8) return "U8";
+    if (v == InferenceEngine::Precision::I4) return "I4";
+    if (v == InferenceEngine::Precision::U4) return "U4";
+    if (v == InferenceEngine::Precision::BIN) return "BIN";
+    if (v == InferenceEngine::Precision::UNSPECIFIED) return "UNSPECIFIED";
+    assert(!"unknown precision");
+    return "undef";
+}
```

Comment:
This function seems to be unnecessary since InferenceEngine::Precision has name() method.

Reply:
nice class. updated. Thank you!